<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomMatrix</class>
 <widget class="QDialog" name="CustomMatrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>710</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix00Box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix01Box">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix02Box">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix10Box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix11Box">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix12Box">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix20Box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix21Box">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="matrix22Box">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="divBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="shiftBox">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>140</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Делитель</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>200</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Смещение</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoCheckBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Автоопределиение делителя</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomMatrix</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomMatrix</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
